source/ShipInfoDisplay.cpp

```diff
@@ -371,6 +375,27 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const PlayerInfo &playe
 		* (1. + attributes.Get("hull heat multiplier")) : 0.;
 	heatTable.push_back(Format::Number(60. * (shieldHeat + hullHeat)));
 
+	// For detailed shipyard view
```

Comment:
The variable could be named in such a way that the comment isnt needed

Reply:
I've opted for infinitePanel (advancedPanel sounds like it has something weird going on), although I would be happy to call it something else instead.

Reply:
~~Infinite panel sounds like a superhero stuff~~

Reply:
How about just scrollingPanel?

Reply:
I've renamed it to scrollingPanel, as no other opinions have been forthcoming.